index.ts

```diff
@@ -1,49 +1,56 @@
-require("dotenv").config();
-const { Client, Intents } = require("discord.js");
-
-const config = { phrases: {} };
-config.phrases.intro = require("./assets/phrases/intro.json");
-config.phrases.welcoming = require("./assets/phrases/welcoming.json");
-
+import { GuildMember, Message, Client, Intents, TextChannel } from "discord.js";
+import "dotenv/config.js";
 const { DISCORD_TOKEN } = process.env;
 
-const client = new Client({
-  intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MEMBERS, Intents.FLAGS.GUILD_MESSAGES],
-});
+interface IPhrases {
+  intro: Array<string>;
+  welcoming: Array<string>;
+}
```

Comment:
Numa review muito rápida, saltou-me à vista o prefixo do I aqui.

Sugeria não usarmos prefixos porque [várias](https://stackoverflow.com/a/41967120/1044644) [razões](https://passionfordev.com/you-dont-need-to-prefix-interfaces-in-typescript-with-i/), para mim uma das principais sendo que do que percebi `interfaces` e `types` em TS são praticamente `interchangeable`, sendo que uma `interface` no seu real sentido é uma abstração para algo, daí me parecer que não me faz sequer muito sentido o prefixo `I` para identificar algo que não é o seu propósito.

O que te parece?

Reply:
Sim, concordo 100%. Sendo assim, tens identificar as interfaces de alguma maneira em específica?

Reply:
Lembro-me de um exemplo que me influenciou bastante nisto.

Um Mercedes é um carro. Um BMW é um carro.

Se fores a um stand pedes um carro, não uma interface de um carro. Eu optaria por não identificar.

Reply:
Concordo. 😀